cpp/test/IceGrid/update/test.py

```diff
@@ -10,13 +10,16 @@ def setupClientSide(self, current):
         current.mkdirs("db/node-1")
         current.mkdirs("db/node-2")
 
-clientProps = lambda process, current: {
-    "NodePropertiesOverride" : current.testcase.icegridnode[0].getPropertiesOverride(current),
-    "IceBoxExe" : IceBox().getCommandLine(current),
-    "IceGridNodeExe" : IceGridNode().getCommandLine(current),
-    "ServerDir" : current.getBuildDir("server"),
-    "TestDir" : "{testdir}"
+
+def clientProps(process, current): return {
+    "NodePropertiesOverride": current.testcase.icegridnode[0].getPropertiesOverride(current),
+    "IceBoxExe": IceBox().getCommandLine(current),
+    "IceGridNodeExe": IceGridNode().getCommandLine(current),
+    "ServerDir": current.getBuildDir("server"),
+    "TestDir": "{testdir}"
 }
 
+
 if isinstance(platform, Windows) or os.getuid() != 0:
-    TestSuite(__file__, [IceGridUpdateTestCase(application=None, client=IceGridClient(props=clientProps))], multihost=False)
+    TestSuite(__file__, [IceGridUpdateTestCase(application=None,
+              client=IceGridClient(props=clientProps))], multihost=False)
```

Comment:
I don't like this formatting very much

```
TestSuite(__file__, 
          [IceGridUpdateTestCase(application=None, client=IceGridClient(props=clientProps))],
          multihost=False)
```
seems more natural

Reply:
For now I think we should just to the defaults. Once all the linting is done we can go back and tweak the style if we really want to.